nml/actions/action0properties.py

```diff
@@ -515,7 +515,23 @@ def aircraft_is_large(value):
     'graphic_flags'  : {'size': 1, 'num': 0x09},
 }
 
-# TODO: Feature 0x06
+#
+# Feature 0x06 (Bridges)
+#
+properties[0x06] = {
+  # 0x08 - year availability (old way)
+  'min_length'              : { 'num': 0x09, 'size': 1 },
+  'max_length'              : { 'num': 0x0A, 'size': 1 },
+  # 0x0B - cost factor (old way)
+  'speed_limit'             : { 'num': 0x0C, 'size': 2, 'unit_type': 'speed', 'unit_conversion': (5000, 1397), 'adjust_value': lambda val, unit: ottd_display_speed(val, 1, unit)},
+  'sprite'                  : { 'num': 0x0D, 'size': 2 },
```

Comment:
This seems wrong, based on my reading of the specs.

Reply:
Hmm... I did it based on [https://newgrf-specs.tt-wiki.net/wiki/Action0/Bridges](https://newgrf-specs.tt-wiki.net/wiki/Action0/Bridges)

What is wrong according to you?

Reply:
prop 0D is not a simple 2 bytes value, it's more complex. See the example at the bottom of the page.

Reply:
Sprites are clearly wrong... Need to find some to generate output, will be able to share full version then.